tests/test_sycl_queue.py

```diff
@@ -1663,6 +1663,7 @@ def test_array_creation_from_dpctl(copy, device):
     assert isinstance(result, dpnp_array)
 
 
+@pytest.mark.usefixtures("suppress_invalid_numpy_warnings")
```

Comment:
Should the fixture be added to `test_from_dlpack_with_dpt` instead?

Reply:
Yes, I have fixed it.
Thank you